src/jdk.javadoc/share/classes/jdk/internal/org/commonmark/internal/util/Html5Entities.java

```diff
@@ -0,0 +1,71 @@
+package jdk.internal.org.commonmark.internal.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Html5Entities {
+
+    private static final Map<String, String> NAMED_CHARACTER_REFERENCES = readEntities();
+    private static final String ENTITY_PATH = "/org/commonmark/internal/util/entities.properties";
```

Comment:
fixed. 

Actually, the `.properties` file is not a Properties file as defined in `java.util.Properties` and is read line-at-a-time with custom code.  To avoid automated compilation of the file by the JDK build system, the file needs to avoid the `.properties` extension.  Currently, using plain old `.txt`.